### PR TITLE
test(monitoring): write tests

### LINKED ISSUE #4: [monitoring] write tests

### PR DESCRIPTION
**Pull request by rofe**
Fix #4

### CODE REVIEW

File: test/testStatuspage.js [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+    description: 'This is a test group',
+    position: 0,
+  };
+
+  before(() => {
+    try {
+      sinon.spy(logger, 'log');
+      sinon.spy(logger, 'error');
+    } catch (e) {
+      // ignore, already wrapped
```
Comment (kptdobe) [inline test/testStatuspage.js]: Not nice ;)
Wouldn't a `sinon.restore()` in the after avoid the exception ? (I am not sure, just asking)

File: test/utils.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+/* eslint-env mocha */
+
+function getTimedPromise(fn, err, time) {
```
Comment (kptdobe) [inline test/utils.js]: Is this something we really have to invent ? Isn't there a standard way to do this ?

Comment (rofe) [reply test/utils.js]: I was wondering the same 😄

File: test/testStatuspage.js (hunk 1 above)
Comment (rofe) [reply test/testStatuspage.js]: Good point, will try

Comment (rofe) [reply test/testStatuspage.js]: Worked, thanks!

File: test/utils.js (hunk 2 above)
Comment (rofe) [reply test/utils.js]: Looks like it's not that far-fetched: https://stackoverflow.com/questions/22707475/how-to-make-a-promise-from-settimeout

**Pull request merged**